tests/system/checks.go

```diff
@@ -31,7 +31,17 @@ func runCheck(t *testing.T, step string, ak *akResult, resp *httpResponse) error
 	}
 }
 
-func checkAKOutput(step string, ak *akResult) error {
+func nextField(text string) (string, string) {
```

Comment:
Please document this non-trivial function. I don't understand what it's doing. A unit test would also be great, since this is an isolated function.

Reply:
done